### PR TITLE
Frontend polkadotjs version bump

### PR DESCRIPTION
**Pull request by saadjhk**
## Issue
*Please replace this line with either a link to the ClickUp issue, or a reference to the GitHub issue.*

## Description
*Please replace this line with info that is not in the ClickUp ticket, eg: "also bumps dependency foo version"*

## Checklist

- [ ] I have updated the cargo docs to reflect changes made by this PR _(if applicable)_
- [ ] I have updated the `book/` to reflect changes made by this PR _(if applicable)_

### CODE REVIEW

File: frontend/apps/picasso/package.json [hunk 1]
```diff
@@ -3,8 +3,8 @@
   "version": "1.0.0",
   "private": true,
   "scripts": {
-    "dev": "node_modules/.bin/next dev",
-    "build": "node_modules/.bin/next build",
```
Comment (easteregg) [inline frontend/apps/picasso/package.json:7]: Pref using  `node_modules/.bin/next` instead of global values which might cause issues with inconsistent build if the user has it installed globally.

File: frontend/apps/pablo/pages/_app.tsx [hunk 2]
```diff
@@ -1,3 +1,8 @@
+import "defi-interfaces/types-lookup.d";
+import "defi-interfaces/augment-api-tx";
+import "defi-interfaces/augment-api-rpc";
+import "defi-interfaces/augment-types";
+
```
Comment (easteregg) [inline frontend/apps/pablo/pages/_app.tsx:5]: We should add these imports to Picasso as well.

File: frontend/apps/pablo/components/Organisms/Wallet/PolkadotConnect.tsx [hunk 3]
```diff
@@ -155,6 +155,48 @@ export const PolkadotConnect: React.FC<{}> = () => {
...
+                <Box sx={{ marginRight: theme.spacing(2) }}>
+                  <Image
+                    src="/networks/polkadot_js.svg"
+                    width="24"
+                    height="24"
+                    alt="Polkadot.js"
+                  />
+                </Box>
+                <Typography variant="button">Nova</Typography>
+              </Button>
```
Comment (easteregg) [inline frontend/apps/pablo/components/Organisms/Wallet/PolkadotConnect.tsx:199]: Looks like this is unrelated to the PR?

File: frontend/apps/picasso/package.json (hunk 1 above)
Comment (saadjhk) [reply frontend/apps/picasso/package.json:7]: should i make it `npx next dev` and `npx next build` respectively

**Review by easteregg — APPROVED** [review]

**Review by andresmechali — APPROVED** [review]

**Comment by saadjhk** [comment]
@mergify refresh
